### PR TITLE
Upgrade to thriftrw-3.0.1

### PR DESCRIPTION
**Pull request by malandrew**

### CODE REVIEW

File: as/thrift.js [hunk 1]
```diff
@@ -47,8 +47,15 @@ function TChannelAsThrift(opts) {
     self.thriftSource = opts.source;
     self.thriftFileName = opts.thriftFileName || 'service.thrift';
     self.spec = new thriftrw.Thrift({
+        entryPoint: opts.entryPoint,
+        idls: opts.idls,
+        thrifts: opts.thrifts,
```
Comment (kriskowal) [inline as/thrift.js]: I don’t recall `thrifts` being an argument.

File: as/thrift.js [hunk 2]
```diff
@@ -47,8 +47,15 @@ function TChannelAsThrift(opts) {
...
         source: self.thriftSource,
-        strict: opts.strict
+        strict: opts.strict,
+        allowFilesystemAccess: opts.allowFilesystemAccess,
+        allowIncludeAlias: opts.allowIncludeAlias,
+        fs: opts.fs,
+        memo: opts.memo
```
Comment (kriskowal) [inline as/thrift.js]: Let’s not expose the memo at this time. It would be harder to remove than to add when we need it.

**Comment by malandrew** [comment]
@kriskowal Can I land this and update the stooge?

**Comment by kriskowal** [comment]
:ok:

**Pull request merged**